app/controllers/PaperlessPreferencesController.scala

```diff
@@ -51,13 +51,9 @@ class PaperlessPreferencesController @Inject()(
                 Some("global.error.BadRequest.heading"),
                 List("global.error.BadRequest.message"))))
         } else {
-          for {
-            managePrefsPartial <- preferencesFrontendPartialService.getManagePreferencesPartial(
-                                   configDecorator.pertaxFrontendHomeUrl,
-                                   Messages("label.back_to_account_home"))
-          } yield {
-            Ok(views.html.preferences.managePrefs(managePrefsPartial.successfulContentOrEmpty))
-          }
+          Future.successful(
+            Redirect(preferencesFrontendService
+              .getManagePreferencesUrl(configDecorator.pertaxFrontendHomeUrl, Messages("label.back_to_account_home"))))
```

Comment:
I'm not sure why we need this service to simply return a URL. If we're going to have a simple redirect then why not simply build the url here and test that we've redirected to the expected url. We don't really need a service to build a redirect url imo.

I'd recommend that if you're happy with renaming the service, to remove it since it isn't a service that returns a Partial (and it a bit out of place in a partials package). Then just change the test to confirm the redirect url rather than a call to this service.

Reply:
I removed partials.PreferncessFrontendService and updated PR